wp-parsely.php

```diff
@@ -188,3 +202,18 @@ function parsely_integrations( $parsely = null ): Integrations {
 
 	return $parsely_integrations;
 }
+
+/**
+ * Instantiates and runs the specified API endpoint.
+ *
+ * @since 3.6.0
+ *
+ * @param string $proxy_class_name The proxy class to instantiate.
+ * @param string $api_proxy_class_name The API proxy class to instantiate and run.
+ */
+function parsely_run_rest_api_endpoint( string $proxy_class_name, string $api_proxy_class_name ): void {
+	$proxy_instance        = new $proxy_class_name( $GLOBALS['parsely'] );
+	$cached_proxy_instance = new Cached_Proxy( $proxy_instance, new WordPress_Cache() );
```

Comment:
I wonder if we could avoid spinning up new instances of this WP cache each time.

```php
static $cache = new WordPress_Cache();
...
$cached_proxy_instance = new Cached_Proxy( $proxy_instance, $cache );
...
```

Reply:
Now tracked in #1210.